packages/catalog-process/test/createEService.test.ts

```diff
@@ -41,6 +42,8 @@ describe("create eservice", () => {
   });
   it("should write on event-store for the creation of an eservice", async () => {
     const isSignalHubEnabled = randomArrayItem([false, true, undefined]);
+    const isDelegable = randomBoolean();
```

Comment:
Use the same used above for `isSignalHubEnabled` to test the case where it's not set
```suggestion
    const isDelegable = randomArrayItem([false, true, undefined]);
```